### PR TITLE
[netstandard] Migrated EditorConfig.Core project to support multiple …

### PR DESCRIPTION
**Pull request by adalon**
…frameworks

Including netstandard, net40 and net45.

TODO: I could not find an easy way to run and update the build targets to generate the editorconfig.nupkg

At least what it's missing is to copy the output of the EditorConfig.Core.csproj (bin\Debug\net40,
bin\Debug\net45 and bin\Debug\netstandard2.0) to their corresponding directories in the output dir.

Fixes issue#2 - Target NetStandard

https://github.com/editorconfig/editorconfig-core-net/issues/2

### CODE REVIEW

**Comment by kzu** [comment]
Would love to see this merged and shipping. It's awesome how moving to SDK-style and multi-targeting makes this soooo much nicer!

💯 @adalon

**Comment by kyle-rader** [comment]
+1 For getting this merged in. Would love to use dotnet core instead of Python (for bootstrapping reasons, at Microsoft).

File: src/EditorConfig.Core/EditorConfig.Core.csproj [hunk 1]
```diff
@@ -1,69 +1,12 @@
...
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFrameworks>net40;net45;netstandard2.0</TargetFrameworks>
```
Comment (kyle-rader) [inline src/EditorConfig.Core/EditorConfig.Core.csproj]: Can you also add `net46`?

Comment (kzu) [reply src/EditorConfig.Core/EditorConfig.Core.csproj]: net46 can readily consume net45 assemblies, not really necessary to target specifically

Comment (kzu) [reply src/EditorConfig.Core/EditorConfig.Core.csproj]: Actually, why even have net45 specifically if there's already net40 @adalon?

**Comment by Mpdreamz** [comment]
Thank you @adalon for this PR! Pardon the looong radio silence on my end.

I am merging this in as is. Will iterate over the packaging scripts afterwards.

@kzu @kyle-rader [as per your discussion](https://github.com/editorconfig/editorconfig-core-net/pull/9#discussion_r245094262) I think i will settle on `net40;netstandard1.3;netstandard2.0` after merging this in. If you have a specific reason for `net46` would love to hear it!

**Pull request merged**